.github/workflows/images-legacy-hotfix-releases.yaml

```diff
@@ -0,0 +1,118 @@
+name: Hot Fix Image Release Build
+
+on:
+  push:
+    branches:
+      - hf/master/**
```

Comment:
I guess when we branch for v1.10, we'll have a bunch of new pieces of infrastructure to configure correctly like this.

Reply:
correct